randovania/monitoring/__init__.py

```diff
@@ -150,8 +150,9 @@ def traces_sampler(sampling_context):
 def client_init() -> None:
     _init(False, "client", exclude_server_name=True)
 
-    sentry_sdk.set_tag("frozen", randovania.is_frozen())
-    sentry_sdk.set_tag("cpu.architecture", platform.machine())
+    global_scope = sentry_sdk.Scope.get_global_scope()
+    global_scope.set_tag("frozen", randovania.is_frozen())
+    global_scope.set_tag("cpu.architecture", platform.machine())
```

Comment:
Thoughts on logging the CPU name too? Seems useful for our performance measuring.

Reply:
I thought that'd be hard but turns out `platform.processor()` exists